cms/apps/media/templates/admin/media/file/change_list.html

```diff
@@ -12,6 +12,18 @@
 
         // Don't assume that the only place this will be used is in a TinyMCE.
         if (!tinymce) {
+          $("div#changelist tr.row1 a, div#changelist tr.row2 a").attr("onclick", null).click(function(event) {
+            // URLs of the form /admin/media/file/1/change/
+            var pk = $(this).attr("href");
+            pk = pk.split('/')
+            pk = pk[4]
+
+            // Prevent default
+            event.preventDefault()
+
+            // Inform the parent page of the selected item
+            window.parent.postMessage(pk, '*');
```

Comment:
`*` should not be used in this case. From MDN:

> Always provide a specific targetOrigin, not *, if you know where the other window's document should be located. Failing to provide a specific target discloses the data you send to any interested malicious site.

In this case it should _always_ be the same origin - use `window.location.origin` instead.